### PR TITLE
FIXES in cmd_anal.c (if else blocks) cons.c (Possible division by zero)

### PR DESCRIPTION
**Pull request by ChD1**
**Checklist**

- [ ] Closing issues: #issue
- [ ] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [radare2book](https://github.com/radareorg/radare2book)

**Description**

### CODE REVIEW

File: libr/cons/cons.c [hunk 1]
```diff
@@ -1268,7 +1268,7 @@ R_API int r_cons_get_column(void) {
 
 /* final entrypoint for adding stuff in the buffer screen */
 R_API int r_cons_memcat(const char *str, int len) {
-	if (len < 0) {
+	if (!str || len < 0) {
```
Comment (trufae) [inline libr/cons/cons.c:1271]: use R_STR_ISEMPTY() so also check if !*str

File: libr/core/cmd_anal.c [hunk 2]
```diff
@@ -6614,29 +6614,37 @@ static void cmd_anal_esil(RCore *core, const char *input) {
...
-		} else if (input[1] == 'j') {
-			cmd_aea (core, 1 + (1<<4), core->offset, r_num_math (core->num, input+2));
-		} else if (input[1] == '*') {
-			cmd_aea (core, 1 + (1<<5), core->offset, r_num_math (core->num, input+2));
-		} else if (input[1] == 'f') {
-			RAnalFunction *fcn = r_anal_get_fcn_in (core->anal, core->offset, -1);
-			if (fcn) {
-				cmd_aea (core, 1, r_anal_function_min_addr (fcn), r_anal_function_linear_size (fcn));
+		switch (input[1]){
+			case '?':
```
Comment (trufae) [inline libr/core/cmd_anal.c:6618]: case must be indented at the same level as the switch statement

**Review by trufae — CHANGES_REQUESTED** [review]
cool!!

File: libr/cons/cons.c (hunk 1 above)
Comment (ChD1) [reply libr/cons/cons.c]: fixed

File: libr/core/cmd_anal.c (hunk 2 above)
Comment (ChD1) [reply libr/core/cmd_anal.c]: fixed

File: libr/cons/cons.c [hunk 3]
```diff
@@ -1268,7 +1268,7 @@ R_API int r_cons_get_column(void) {
 
 /* final entrypoint for adding stuff in the buffer screen */
 R_API int r_cons_memcat(const char *str, int len) {
-	if (len < 0) {
+	if (R_STR_ISEMPTY(str) || len < 0) {
```
Comment (trufae) [inline libr/cons/cons.c:1271]: respect indentation rules  (missing space before `(`)
```suggestion
	if (R_STR_ISEMPTY (str) || len < 0) {
```

**Pull request merged**